processes/release_processes/README.MD

```diff
@@ -81,6 +85,7 @@ Using the Swagger API, submit the workflow to Cromwell along with the inputs fil
 * Look [in Travis](https://app.travis-ci.com/github/broadinstitute/cromwell/branches) for the release tag build that will publish Docker images for the new version.
 * Let `#ftfy-private` know that it's okay to resume merges to `develop`.
 * Announce release in `#dsp-batch`, set expectations about when the new version will be available in Terra.
+* It will take about an additional hour for the Docker image to build in Travis before its tag appears on the [Cromwell Docker Hub page](https://hub.docker.com/r/broadinstitute/cromwell/tags).
```

Comment:
TOL would it be helpful to say which sub-build is responsible for building and pushing the image?

Reply:
Per the build scripts, the magic combination is [hotfix branch](https://github.com/broadinstitute/cromwell/blob/a22bd9baba075c688e4222067db2a5ebdcf2f46b/src/ci/bin/test.inc.sh#L1629) with [empty CROMWELL_BUILD_SBT_INCLUDE](https://github.com/broadinstitute/cromwell/blob/a22bd9baba075c688e4222067db2a5ebdcf2f46b/src/ci/bin/test.inc.sh#L1619), which through [this hop](https://github.com/broadinstitute/cromwell/blob/a22bd9baba075c688e4222067db2a5ebdcf2f46b/src/ci/bin/test.inc.sh#L346) works out to be the ["catchall" sbt build](https://github.com/broadinstitute/cromwell/blob/fdedcbb132dfa5edb2d5777ae57a33beb797538b/.travis.yml#L101-L102).